packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -121,6 +147,31 @@ export class CustomAppComponent implements AfterViewInit {
         alert(`${menuItemText} selected`);
     }
 
+    public onDynamicSelectFilterInput(e: Event): void {
+        const event = e as CustomEvent<SelectFilterInputEventDetail>;
+        const filter = event.detail.filterText;
+        if (filter.length > 0) {
+            window.clearTimeout(this.dynamicSelectFilterTimeout);
+            this.dynamicSelect.loadingVisible = true;
+            this.dynamicSelectFilterTimeout = window.setTimeout(() => {
+                // do your custom filtering here
+                const filteredItems = this.availableSelectItems.filter(item => item.first.concat(item.last).toLowerCase().includes(filter.toLowerCase()));
```

Comment:
Do we want to encourage using the diacritic insensitive filter by using it here?

Reply:
I'm not entirely worried about this, as often will be the case the actual filtering mechanism will be in a backend implementation that can not use that filter. That is, the code in question is likely to be replaced with an http request.